packages/clay-layout/src/Container.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import React from 'react';
+
+interface IProps extends React.HTMLAttributes<HTMLDivElement> {
+	/**
+	 * Adds `.container-fluid` class to create a fluid container that
+	 * doesn't expand beyond a set width
+	 */
+	fluid?: boolean;
+
+	/**
+	 * Adds the `.container-form-${formSize}` class to properly space
+	 * between application controls and the form. This class only modifies
+	 * the padding on the container.
+	 */
+	formSize?: 'sm' | 'md' | 'lg' | 'xl';
+
+	/**
+	 * Adds `.container-fluid-${size}` class to set max width on container.
+	 */
+	fluidSize?: 'sm' | 'md' | 'lg' | 'xl';
```

Comment:
We just [called this size](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ContainerTag.java#L160) in our JSP tag since it seems to be the basic function of the Container, so why the prefix?

We also default to `xl` since it's by far the main usage we saw across the board... might want to consider it here too

Reply:
I used the prefix since I didn't want people to think `size` could correlate to `container-${size}`. I thin by making it explicitly named `fluid` its apparent that its for the `container-fluid` class

Reply:
I think the correct name properties should be `fluidMaxSize` since we only use classes `container-fluid-max-*breakpoint*` to create all Lexicon grids they need. On the other hand, I would recommend forgetting all rest of ways to create container size limit because there are a lot which probably a final customer could use but we do not need

Reply:
So... now that we have `container-fluid`... maybe this could be:

`fluid: boolean | ['sm', 'md', 'lg', 'xl']`?